target/scripts/helpers/utils.sh

```diff
@@ -63,3 +63,48 @@ function _reload_postfix
   _adjust_mtime_for_postfix_maincf
   postfix reload
 }
+
+# Replaces values in configuration files given a set of specific environment
+# variables. The environment variables follow a naming pattern, whereby every
+# variable that is taken into account has a given prefix. The new value in the
+# configuration will be the one the environment variable had at the time of
+# calling this function.
+#
+# @param ${1} = prefix for environment variables
+# @param ${2} = file in which substitutions should take place
+#
+# ## Example
+#
+# If you want to set a new value for `readme_directory` in Postfix's `main.cf`,
+# you can set the environment variable `POSTFIX_README_DIRECTORY='/new/dir/'`
+# (`POSTFIX_` is an arbitrary prefix, you can choose the one you like),
+# and then call this function:
+# `_replace_by_env_in_file 'POSTFIX_' 'PATH TO POSTFIX's main.cf>`
+function _replace_by_env_in_file
+{
+  if [[ -z ${1+set} ]]
+  then
+    _log 'warn' "(ldap.sh:_replace_in_file) missing first argument"
```

Comment:
Note that these logs you have several of reference `ldap.sh:_replace_in_file`, I assume that needs to be updated to new file/method name?

Is a warning correct, or should it be failing hard since this is at start-up? There's `dms_panic` method to do a similar failure with context of where the failure is happening (_I've used it in `ssl.sh` helper_).